ingestion/monitoring/freshness.py

```diff
@@ -0,0 +1,80 @@
+import os
+import sys
+import json
+import logging
+import operator
+from typing import Any
+from collections import defaultdict
+
+import boto3
+import requests
+
+DEFAULT_INSTANCE = "https://data.covid-19.global.health"
+
+
+def fetch_sources(api_key, instance):
+    logging.info(f"Fetching sources from {instance}")
+    res = requests.get(
+        f"{instance}/api/sources?limit=100",
+        headers={
+            "Content-Type": "application/json",
+            "X-API-Key": api_key,
+        },
+    )
+    return res.json() if res.status_code == 200 else None
```

Comment:
If 4XX or 5XX log the response?

Reply:
Agreed

Reply:
Also consider what to do if there are more than 100 sources, I know it looks like that won't ever be a problem with the covid-19 data sources but we should at least error out if we're not going to handle it, for when it eventually happens on this/another deployment.

Reply:
thanks, added paging support